Formula/subversion.rb

```diff
@@ -164,12 +167,18 @@ def install
       --without-apache-libexecdir
       --without-berkeley-db
       --without-gpg-agent
-      --enable-javahl
       --without-jikes
       PERL=#{perl}
       PYTHON=#{Formula["python@3.10"].opt_bin}/python3
       RUBY=#{ruby}
     ]
+    if openjdk
+      args.unshift "--with-jdk=#{Formula["openjdk"].opt_prefix}",
+                   "--enable-javahl"
+    end
+
+    # preserve compatibility with macOS 12.0–12.2
+    args.unshift "--enable-sqlite-compatibility-version=3.36.0" if MacOS.version >= :monterey
```

Comment:
```suggestion
    args.unshift "--enable-sqlite-compatibility-version=3.36.0" if MacOS.version == :monterey
```

For macOS 13, we won't need 3.36 compatibility.